### PR TITLE
[onert/test] Remove nnpackage and modelfile options

### PR DESCRIPTION
**Pull request by hseok-oh**
This commit removes "--nnpackage" and "--modelfile" options from `onert_run` and `onert_train` test drivers. 
They can be handled on last parameter without option automatically.

ONE-DCO-1.0-Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

### CODE REVIEW

File: tests/tools/onert_train/README.md [hunk 1]
```diff
@@ -18,17 +18,17 @@ sudo apt install -y libhdf5-dev libboost-program-options-dev
 
 ## Usage
 
-You could train your model using the command like below.  
+You could train your model using the command like below.
 
 ```bash
 onert_train \
 --path [circle file or nnpackage] \
```
Comment (jyoungyun) [inline tests/tools/onert_train/README.md:25]:
```suggestion
```

File: tests/tools/onert_train/README.md [hunk 2]
```diff
@@ -18,17 +18,17 @@ sudo apt install -y libhdf5-dev libboost-program-options-dev
...
 --load_expected:raw [training output data] \
---batch_size 32 \ 
+--batch_size 32 \
 --epoch 5 \
 --optimizer 1 \             # sgd
---learning_rate 0.01 \   
+--learning_rate 0.01 \
 --loss 2 \                  # cateogrical crossentropy
 --loss_reduction_type 1     # sum over batch size
 --num_of_trainable_ops 30    # number of operations to be trained from the back
```
Comment (jyoungyun) [inline tests/tools/onert_train/README.md:34]:
```suggestion
--num_of_trainable_ops 30 \    # number of operations to be trained from the back
[circle file or nnpackage]
```

File: tests/tools/onert_train/README.md [hunk 3]
```diff
@@ -18,17 +18,17 @@ sudo apt install -y libhdf5-dev libboost-program-options-dev
...
 --load_input:raw [training input data] \
 --load_expected:raw [training output data] \
---batch_size 32 \ 
+--batch_size 32 \
 --epoch 5 \
 --optimizer 1 \             # sgd
---learning_rate 0.01 \   
+--learning_rate 0.01 \
 --loss 2 \                  # cateogrical crossentropy
 --loss_reduction_type 1     # sum over batch size
```
Comment (jyoungyun) [inline tests/tools/onert_train/README.md:33]: (optional)
```suggestion
--loss_reduction_type 1 \    # sum over batch size
```

File: tests/tools/onert_train/README.md [hunk 4]
```diff
@@ -70,28 +70,28 @@ $ tree out
...
+You could convert tf/tflite/onnx model file into circle file using [`onecc`](https://github.com/Samsung/ONE/tree/master/compiler/one-cmds). <br/>
+If you start with tensorflow code, you could first save it as saved format and then convert it to a circle file by using `onecc`.
 
 <!--TODO : Add how to inject training parameter in the circle model -->
- 
+
 ### Run onert_train
 Now you're ready to run `onert_train`. <br/>
-Please pass your model file to `--modelfile` and data files to `--load_input:raw` and `--load_expected:raw`. <br/>
+Please pass your model file as argument and data files with `--load_input:raw` and `--load_expected:raw` option. <br/>
```
Comment (jyoungyun) [inline tests/tools/onert_train/README.md:81]:
```suggestion
Please pass your model file as argument and data files with `--load_input:raw` and `--load_expected:raw` options. <br/>
```

**Review by jyoungyun — COMMENTED** [review]
Looks good to me except for a few suggestions. :)

**Review by ragmani — APPROVED** [review]
LGTM

**Review by chunseoklee — APPROVED** [review]

**Review by glistening — APPROVED** [review]
LGTM
